ui-tests/src/test/resources/features/integrations/email.feature

```diff
@@ -0,0 +1,118 @@
+    And select "Data Mapper" integration step
+    And create data mapper mappings
+      | company | content |
+    And scroll "top" "right"
+    And click on the "Done" button
+    Then check visibility of page "Add to Integration"
+
+    # Publish integration
+    When click on the "Save" button
+    And set integration name "email-send-qe-integration"
+    And publish integration
+    Then Integration "email-send-qe-integration" is present in integrations list
+    And wait until integration "email-send-qe-integration" gets into "Running" state
+
+    # Wait integration to do a first periodic query and check that email has arrived
+    When sleep for "10000" ms
+    Then Integration "email-send-qe-integration" is present in integrations list
+    Then check that email from "jbossqa.fuse.email@gmail.com" with subject "syndesis-tests" and text "Red Hat" exists
+    
+  @email-receive  
+  Scenario: Receive Email throught IMAP with SSL
+    
+    When click on the "Create Integration" button to create a new integration
+    Then check visibility of visual integration editor
+    And check that position of connection to fill is "Start"
+
+    # First connection is email receive (imap or pop3), by default it only fetches unread emails
+    When select the "Receive Email with IMAP QE" connection
+    Then select "Receive Email" integration action
+    And fill in values
+      | delay          | 30 |
+      | maximum emails | 10 |
+    And click on the "Done" button
+    Then check that position of connection to fill is "Finish"
+
+    # Second connection is insert into TODO table in database
+    When select the "PostgresDB" connection
+    And select "Invoke SQL" integration action
+    And fill in invoke query input with "insert into todo(task, completed) values(:#task, 0)" value
+    And click on the "Next" button
+    Then check visibility of page "Add to Integration"
+
+    # Two integration steps: split and data mapper, which maps email content to 'task' field in TODO table
+    When add integration step on position "0"
+    And select "Split" integration step
+    Then check visibility of page "Add to Integration"
+
+    When add integration step on position "1"
+    And select "Data Mapper" integration step
+    And create data mapper mappings
+      | content | task |
+    And scroll "top" "right"
+    And click on the "Done" button
+    Then check visibility of page "Add to Integration"
+
+    # Publish integration
+    When click on the "Save" button
+    And set integration name "email-receive-qe-integration"
+    And publish integration
+    Then Integration "email-receive-qe-integration" is present in integrations list
+    And wait until integration "email-receive-qe-integration" gets into "Running" state
+
+    # Send email to connector, wait for it to be received and check that it's content got into database
+    When send an e-mail to "jbossqa.fuse.email@gmail.com"
+    And sleep for "60000" ms
```

Comment:
following step has already dynamic wait set for 60s, is this sleep necessary?

Reply:
seems like it's not necessary. it exists in gmail scenario, which I partly used as steps reference, so I didn't think twice about this one.
will remove, thank you!